Leetcode/aoc_2015_d5.py

```diff
@@ -33,9 +33,11 @@ def nice_string(str_list: list) -> int:
 		# it'd require a nested loop and a variable. This way I can just continue the current loop.
 		if "ab" in s or "cd" in s or "pq" in s or "xy" in s:
 			continue
-		for i in range(len(s)):
+
+		for i in range(len(s) - 1):
 			if i < len(s) - 1 and s[i + 1] == s[i]:
```

Comment:
Loop bounds mean the first half of the conditional isn't needed anymore

```suggestion
			if s[i + 1] == s[i]:
```

Reply:
Whoops, I didn't mean to include that in the conditional.